docs/development/release.rst

```diff
@@ -6,30 +6,47 @@
 How to make a Gammapy release
 *****************************
 
-This page contains step-by-step instructions how to make a Gammapy release. We mostly follow the
-`Astropy release instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#release-procedure-for-the-astropy-core-package>`__
-and just lists the additional required steps.
+This page contains step-by-step instructions for how to make a Gammapy release. The general procedure can
+be broken down into three major steps:
+
+
+* Feature freeze: freeze the core package features, a number of weeks before the release
+* Release candidate: proposed feature release which should be tested, a few weeks before the final release
+* Final release: a final release for the version of the feature release
 
 
 Feature Freeze and Branching
 ----------------------------
 
-#. Follow the `Astropy feature freeze and branching instructions <https://docs.astropy.org/en/latest/development/maintainers/releasing.html#start-of-a-new-release-cycle-feature-freeze-and-branching>`__.
-   Instead of updating the ``whatsnew/<version>.rst`` update the ``docs/release-notes/<version>.rst``.
-#. Update the entry for the feature freeze in the `Gammapy release calendar <https://github.com/gammapy/gammapy/wiki/Release-Calendar>`__.
+#. Add a new milestone to the GitHub issue tracker for the version ``v<version>.x`` (this will also be used for the
+   next bugfix release). Also create a ``backport-v<version>.x`` label.
+#. Update your local ``main`` branch to the latest from remote::
+
+    git fetch upstream --tags --prune
+    git checkout -B main upstream/main
+
+#. Create a new branch with the name of the version::
+
+    git branch v<version>.x
+
+#. Stay on the ``main`` branch and make a copy and update the ``docs/release-notes/<version>.rst``
+#. Commit the changes and push to GitHub main.
+#. Update the entry for the feature freeze in the
+   `Gammapy release calendar <https://github.com/gammapy/gammapy/wiki/Release-Calendar>`__.
 
 
 Releasing the first major release candidate
 -------------------------------------------
 
-A few days before the planned release candidate:
+**A few days before the planned release candidate:**
 
 #. Fill the changelog ``docs/release-notes/<version>.rst`` for the version you are about to release.
 #. Update the author list manually in the  ``CITATION.cff``.
 #. Open a PR including both changes and mark it with the ``backport-v<version>.x`` label.
-   Gather feedback from the Gammapy user and dev community and finally merge and backport to the ``v<version>.x`` branch.
+   Gather feedback from the Gammapy user and dev community and finally merge and backport to the
+   ``v<version>.x`` branch.
```

Comment:
Maybe those steps should be moved to the feature freeze. It takes a lot of time to get all feedback on changelog and author list.

Reply:
I think this time we took longer because we forgot this step at the release candidate stage, right?

Reply:
no, it was started even before the feature freeze actually...